pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -65,7 +65,9 @@ type GCECloud struct {
 	service           *compute.Service
 	containerService  *container.Service
 	projectID         string
-	zone              string
+	region            string
+	instanceZone      string
+	managedZones      []string
```

Comment:
How do you dynamically add zones to this list after the cluster is running? The way you explained creating an ubernetes lite cluster at the SIG meeting was that you'd set up a cluster normally and then re-run the kube-up script adding additional zones. That doesn't seem to jive with pre-propulating a fixed list of zones that are being "managed". 

@brendandburns